string pcloud_filename
string image_filename
string mask_filename
---
omron_cad_matching/SearchResult search_result
bool success
